Helper/Data.php

```diff
@@ -104,6 +107,43 @@ public function getReachConfig($code, $storeId = null)
         return $this->getConfigValue(self::XML_PATH_REACH .'reach_gointerpay/'. $code, $storeId);
     }
 
+    /**
+     * Get Transport Mode for DHL DHL_TRANSPORT_MODE_PATH
+     *
+     * @return void
+     */
+    public function getTransportMode() {
+        return $this->getConfigValue(self::DHL_TRANSPORT_MODE_PATH);
+    }
+
+    /**
+     * Get End Use DHL_END_USE_PATH
+     *
+     * @return string|null
```

Comment:
since there will always be a default value, is it true that these functions will return null?

Reply:
Correct. These values are from a drop down list that must contain a value from the list. Null would be applicable if text field or text entry box.